release.yaml

```diff
@@ -30,9 +30,10 @@ longhorn-crd:
   - 102.3.3+up1.5.5
   - 103.2.3+up1.5.5
 harvester-cloud-provider:
-  - 104.0.0+up0.2.3
   - 102.0.2+up0.2.3
   - 103.0.1+up0.2.3
+  - 104.0.0+up0.2.3
```

Comment:
Both these versions will be released in `v2.9.0`?

Reply:
@w13915984028, I think we can remove `102.0.2+up0.2.3` and `103.0.1+up0.2.3`. WDYT? Thank you.

Reply:
Need to hear advise from @lucasmlp, thanks.

Normally, a Chart for old Rancher version, e.g. `102.0.2+up0.2.3` for Rancher v2.7, should/not also be visible&available on new Rancher version v2.9?

Our past PRs were adding a new Harvester related chart to all mainstream Rancher version v2.7, v2.8, v2.9, but not removing any existing chart versions.

Reply:
And according to the naming rules, we already have https://github.com/rancher/charts/blob/dev-v2.9/charts/harvester-cloud-provider/104.0.0%2Bup0.2.3/values.yaml,  

This PR seems should name it as `104.0.1+up0.2.3`. @FrankYang0529